rush/rush/src/utilities/InstallManager.ts

```diff
@@ -45,6 +45,16 @@ export enum InstallType {
   UnsafePurge
 }
 
+interface IImplicitVersionList {
```

Comment:
>IImplicitVersionList [](start = 10, length = 20)

We definitely need some JSDoc for these.  Nobody besides you is going to know what they do :-)